### ISSUE TITLE
Mix Components do not pipe output or throw errors

### ISSUE THREAD

**Issue by frankfava** [laravel-mix/laravel-mix#3194]
-   Laravel Mix Version: 6.0.39
-   Node Version: v16.13.1
-   NPM Version: v8.1.2
-   OS: MacOS Big Sur 11.4 (20F71)

### Description:

Mix components do not show the output from the included WebPack plugins.

I believe this is due to the **spawn method** (https://github.com/laravel-mix/laravel-mix/blob/bc9063b808bf2689936222469c160b9f8d1f6838/bin/cli.js#L100) as it has the **incorrect syntax**
https://nodejs.org/docs/latest-v16.x/api/child_process.html#child_processspawncommand-args-options

`spawn(command[, args][, options])`

Mix passes a complete string as the first argument and then the options as the 2nd and not the third argument.

This means that **NO OUTPUT** is shown from the plugins, including any thrown errors

**Comment by thecrypticace** [comment]
**It has nothing to do with the invocation of spawn.**

It is correct as written. Options can be passed as the 2nd argument when there are no additional arguments.

The real reason is because output from child compilers is suppressed via stats — which isn't ideal and we should fix.

---

Some information about how to interpret the signatures from the npm docs:

The documented signature for spawn is `spawn(command[, args][, options])` which means the following signatures are valid:
- `spawn(command)`
- `spawn(command, args)`
- `spawn(command, options)`
- `spawn(command, args, options)`

Basically:
1. `command` is required as the first argument in all situations.
2. `args` and `options` are both optional and can be the 2nd argument.
3. Passing `args` as an option means `options` MUST come after it.

If the signature was `spawn(command[, args[, options]])` then options could not be the second argument.

**Comment by frankfava** [comment]
Ah thank you @thecrypticace! Learn something new everyday 💡 

Is there a way to modify the stats varaible on the fly?

**Comment by thecrypticace** [comment]
Try this:
```js
mix.override(config => {
  // Configure webpack to show basically everything
  config.stats = {
    all: true,
  }
})
```

**Comment by frankfava** [comment]
Cheers @thecrypticace 

That didn't solve the problem but it did point me in the right direction.

The plugin that i was using was utilising the Webpack Logger Interface (https://webpack.js.org/api/logging/) so i needed to override this back to the default of `info`

```
mix.override(config => {
  // Configure webpack to show basically everything
  config.stats = 'minimal'
  
  // Change Logger Interface level
  config.infrastructureLogging = { level: 'info' }
})
```
